pom.xml

```diff
@@ -1767,6 +1767,12 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.ratis</groupId>
+                <artifactId>ratis-thirdparty-misc</artifactId>
+                <version>0.7.0</version>
```

Comment:
Can you add a property for this?  e.g. `${dp.ratis.version}`.  See below for Kafka.

Reply:
The thirddparty package is available at a different version than the ratis version. dp.ratis.version already exists for all the other ratis properties.